docs/source/examples/convert_notebooks.py

```diff
@@ -0,0 +1,21 @@
+"""
+This utility script converts all of the notebooks in the /docs/source/examples
+directory into python scripts in the /examples/ directory, to avoid having to
+modify the same code in both locations.
+"""
+
+import os
+from menelaus.datasets.make_example_data import find_git_root
```

Comment:
If working, this appears a lot simpler than the jupyter-execution unit test -- can we import what we need there, too, and use this as a pre-step before performing the actual unit test?

Reply:
@tms-bananaquit the former; I'm wondering if we can refactor the `run_notebook` test to look like this script, since it's similar in function. But it may not be worth messing with as part of this PR -- feel free to ignore.